javascript/chronometer.js

```diff
@@ -1,26 +1,33 @@
 class Chronometer {
-  constructor() {
-    // ... your code goes here
-  }
-  startClick(callback) {
-    // ... your code goes here
-  }
+  constructor() {  //should not receive any arguments
+    this.currentTime = 0; //stablish the currentTime property
+    this.intervalId = 0; //Stablish the intervalID property
+  }
+  startClick(callback) { //if every second currentTime increment by one, every 3 seconds it will increment by 3
+    let intervalId = setInterval(() => {
```

Comment:
You are saving the intervalId in a simple variable, but not in the class property (this.intervalId)